jaypeasee/national-parkfinder
# remove activities from each park object

## Is this a fix or a feature?
Fix

## What is the change?
Removing `activities` property from each park object in `ParkData.js`

## What does it fix?
Makes the data a bit more minimal.

## Where should the reviewer start?
`ParkData.js`

## How should this be tested?
When you go to map through this, check if there are errors.

@saraho1123
@jaypeasee
@baileydunning
